eox_tenant/views.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""The generic views for the eox-tenant plugin project"""
+
+from __future__ import unicode_literals
+
+import json
+from os.path import dirname, realpath
+from subprocess import check_output, CalledProcessError
+
+from django.http import HttpResponse
+from django.utils import six
+
+
+import eox_tenant
+
+
+def info_view(request):  # pylint: disable=unused-argument
```

Comment:
@felipemontoya  this is a view that anyone can see, do we want that behavior? or maybe we should restrict this information

Reply:
Currently for the eox-core view it is just like this. Anyone can view it.

There is an inherent risk if one of our versions has a bug or known vulnerability and it is not yet patched for the instance. But we accepted this for eox-core I don't see why we can't accept it for tenant.
Now, If you know about a recommended practice not to do it this way, please speak up and we can change this behaviour